Formula/emacs-plus@28.rb

```diff
@@ -36,6 +36,7 @@ class EmacsPlusAT28 < EmacsBase
   depends_on "autoconf" => :build
   depends_on "gnu-sed" => :build
   depends_on "gnu-tar" => :build
+  depends_on "grep" => :build
```

Comment:
💯 was busy today and haven't read all the messages yet, but this is what I wanted to propose after seing your recommendation to `brew install grep`.